tests/integration/test_fim/test_files/test_report_changes/test_diff_size_limit_configured.py

```diff
@@ -89,29 +102,27 @@
 # Variables
 
 wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
-test_directories = [os.path.join(PREFIX, 'testdir1')]
-directory_str = ','.join(test_directories)
-test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
-configurations_path = os.path.join(test_data_path, 'wazuh_conf.yaml')
-testdir1 = test_directories[0]
-DIFF_LIMIT_VALUE = 2
+test_directories = [os.path.join(PREFIX, TEST_DIR_1)]
```

Comment:
Why use test_directories as an array, if only One test path will be used? And it won't be passed as an iterable on configuration, but it's hardcoded to only use the first in the array?
Better change it to a simple string instead of an array for legibility.

Reply:
Done in commit [543b1](https://github.com/wazuh/wazuh-qa/pull/2417/commits/543b1358139da987fd41ac84aec5a0ee9ba4f24b)